socorro/external/crashstorage_base.py

```diff
@@ -63,7 +64,7 @@ def as_file_dumps_mapping(self, crash_id, temp_path, dump_file_suffix):
         """convert this MemoryDumpMapping into a FileDumpsMappng by writing
         each of the dump to a filesystem."""
         name_to_pathname_mapping = FileDumpsMapping()
-        for a_dump_name, a_dump in self.iteritems():
```

Comment:
Aha, this one looks particularly tricky (since it's a dict subclass), nice work!